packages/session-replay-browser/src/config/remote-config.ts

```diff
@@ -0,0 +1,121 @@
+import { BaseTransport } from '@amplitude/analytics-core';
+import { ServerZone, Status } from '@amplitude/analytics-types';
+import { UNEXPECTED_ERROR_MESSAGE } from '../messages';
+import { SessionReplaySessionIDBStore as AmplitudeSessionReplaySessionIDBStore } from '../typings/session-replay';
+import {
+  SessionReplayLocalConfig as ISessionReplayLocalConfig,
+  SessionReplayRemoteConfig as ISessionReplayRemoteConfig,
+  SessionReplayRemoteConfigFetch as ISessionReplayRemoteConfigFetch,
+  SamplingConfig,
+  SessionReplayRemoteConfigAPIResponse,
+} from './types';
+
+const UNEXPECTED_NETWORK_ERROR_MESSAGE = 'Network error occurred, session replay remote config fetch failed';
+const SUCCESS_REMOTE_CONFIG = 'Session replay remote config successfully fetched';
+const MAX_RETRIES_EXCEEDED_MESSAGE = 'Session replay remote config fetch rejected due to exceeded retry count';
+export const REMOTE_CONFIG_SERVER_URL = 'https://sr-config.amplitude.com/config';
```

Comment:
We'll also want to use the CDN URL, but definitely TODO on the naming though I imagine this will stick.

```suggestion
export const REMOTE_CONFIG_SERVER_URL = 'https://sr-client-cfg.amplitude.com/config';
```